stl/inc/functional

```diff
@@ -656,26 +656,21 @@ struct _Unforced { // tag to distinguish bind() from bind<R>()
 };
 
 // helper to give INVOKE an explicit return type; avoids undesirable Expression SFINAE
-template <class _Rx, bool = is_void_v<_Rx>>
-struct _Invoker_ret { // selected for _Rx being cv void
-    template <class _Fx, class... _Valtys>
-    static _CONSTEXPR20 void _Call(_Fx&& _Func, _Valtys&&... _Vals) noexcept(_Select_invoke_traits<_Fx,
-        _Valtys...>::_Is_nothrow_invocable::value) { // INVOKE, "implicitly" converted to void
-        _STD invoke(static_cast<_Fx&&>(_Func), static_cast<_Valtys&&>(_Vals)...);
-    }
-};
-
 template <class _Rx>
-struct _Invoker_ret<_Rx, false> { // selected for all _Rx other than cv void and _Unforced
+struct _Invoker_ret {
     template <class _Fx, class... _Valtys>
     static _CONSTEXPR20 _Rx _Call(_Fx&& _Func, _Valtys&&... _Vals) noexcept(_Select_invoke_traits<_Fx,
-        _Valtys...>::template _Is_nothrow_invocable_r<_Rx>::value) { // INVOKE, implicitly converted to _Rx
-        return _STD invoke(static_cast<_Fx&&>(_Func), static_cast<_Valtys&&>(_Vals)...);
+        _Valtys...>::template _Is_nothrow_invocable_r<_Rx>::value) { // INVOKE, implicitly converted

```

Comment:
the noexcept specification differs a bit. did you check that `_Select_invoke_traits<_Fx,
        _Valtys...>::_Is_nothrow_invocable::value ==_Select_invoke_traits<_Fx,
        _Valtys...>::template _Is_nothrow_invocable_r<void>::value `

Reply:
Good point, added `_STL_INTERNAL_STATIC_ASSERT`

Reply:
Also manually checked, they ae the same.